packages/office-ui-fabric-react/src/components/ActivityItem/examples/ActivityItem.Basic.Example.tsx

```diff
@@ -5,7 +5,7 @@ import { getStyles, IActivityItemExampleStyleProps, IActivityItemExampleStyles }
 import { Icon } from '../../../Icon';
 import { Link } from '../../../Link';
 
-export class ActivityItemBasicExample extends React.Component<React.Props<ActivityItemBasicExample>, {}> {
+export class ActivityItemBasicExample extends React.Component<React.ClassAttributes<ActivityItemBasicExample>, {}> {
```

Comment:
What do you think about this type of changes @JasonGore?

Reply:
`React.Props` is: 
```tsx
    interface Props<T> {
        children?: ReactNode;
        key?: Key;
        ref?: Ref<T>;
    }
```

While `React.ClassAttributes` is:

```tsx
    interface ClassAttributes<T> extends Attributes {
        ref?: Ref<T>;
    }
    interface Attributes {
        key?: Key;
    }
```

So the significant difference here is that `children` will no longer be typed in props. But I can't think of a valid (or at least currently "correct") use of `children` via props directly in either JSX or when calling `createElement`. If there are no TypeScript errors, it should be fine.

@cliffkoh Can you think of any previous or existing valid methods for passing in children via props instead of using the third arg in `createElement` or as bounded elements in JSX?

Reply:
It seems to me the real main impact of this change is that you can no longer do this.props.children, which would be a problem, isn't it?

Reply:
Yeah.. I can't believe that didn't come through as a TS error though unless none of these components are using them. @khmakoto Can you make sure one of these changed components is using props.children (or test on a component that uses props.children)? Thanks

Reply:
Sure, let me check on a component that does use props.children and I'll get back to you with an answer.

Reply:
Ok, it turns out that React's `Component` type appends `children` to the props type. 

```tsx
    class Component<P, S> {
        ...
        props: Readonly<{ children?: ReactNode }> & Readonly<P>;
        ...
    }
```
We've tested that in changing `Layer` type to `ClassAttributes` that `children` was still accessible due to this typing.